content/guides/process.md

```diff
@@ -7,127 +7,71 @@ beta: true
 
 ### Introduction
 
-The new Datadog Process Monitor allows for real time process-level visibility. After installing a small secondary agent, information from a host’s /proc file system will be streamed into Datadog and presented in a table at [https://app.datadoghq.com/process](https://app.datadoghq.com/process).
+The new Datadog Process Monitor allows for real-time process-level visibility.  After configuring the Agent, information from a host’s /proc file system will be streamed into Datadog and presented in a table at [https://app.datadoghq.com/process](https://app.datadoghq.com/process).
 
-The table can be searched and filtered and processes can be aggregated on tags associated with the hosts that they are running on.
+The table can be searched and filtered, and processes can be aggregated on tags associated with the hosts that they are running on.
 
 Each row in the table can be expanded for more contextual information about an individual process.
 
 Checking “show summary graphs” at the top of the page will graph metrics for all processes that are shown in the table.
 
 {{< img src="process/live_process_preview.png" >}}
 
-### One-step Installation
+### Configure the Datadog Agent to collect Live Process data
```

Comment:
Should we point to the agent installation page in the app are a prerequisite just in case they come to this page directly?